pipelines/resources/tekton/task/func-deploy/0.1/func-deploy.yaml

```diff
@@ -24,7 +24,29 @@ spec:
       description: The workspace containing the function project
   steps:
   - name: func-deploy
-    image: "registry.redhat.io/openshift-serverless-1/kn-plugin-func-rhel8:latest"
+    image: "registry.access.redhat.com/ubi8/ubi:8.6"
```

Comment:
@jrangelramos Just curious, this task is for the tests or customer will use this too? if this is for customers too then we need `registry.redhat.io` instead of `registry.access.redhat.com`.

Reply:
Hi @Kaustubh-pande , yes this is for our customer.

Reply:
I think you are not right on this @Kaustubh-pande. This is something that customers will execute, right. But, the same is for Knative services or simple pods running on OpenShift. We even advise the in the docs to use ubi as a base image for their apps.

Reply:
I guess it is recommended for customers use `registry.redhat.io`, `registry.access.redhat.com` to provide unauthenticated access, at some point in the future it will decommission as well 

- https://access.redhat.com/RegistryAuthentication#learning-more-about-red-hat-registry-9
- https://source.redhat.com/groups/public/sbr_containers/cee_containers_sbr_wiki/terms_based_authenticated_registry_faq

Maybe I was wrong here but it is good to verify once with Naina or rajesh.

Reply:
@nainaz ^^^

Reply:
We came up with different, better solution, of using client image directly.